spec/models/user_spec.rb

```diff
@@ -1,47 +1,58 @@
-require 'support/notifications_service'
-
 describe User do
-  include_examples 'notifications service'
-
   context 'associations' do
     it { should have_many(:ips) }
     it { should belong_to(:organisation) }
   end
 
-  describe '#update_details' do
-    let(:user) { create(:user, password: 'password') }
-    context 'when passwords match' do
-      let(:params) { { password: '123456', password_confirmation: '123456', name: 'bob' } }
+  context 'validations' do
+    it { should validate_presence_of(:name).on(:update) }
 
-      it 'should set the users password' do
-        expect { user.update_details(params) }.to change(user, :password)
-        expect(user.errors.empty?).to eq(true)
+    describe "password and password confirmation must match" do
```

Comment:
I'd like to change these descriptions slightly, as the nesting means they read a little weirdly to me. I could get back to you with the suggested edit?

It's not a huge deal if you want to press on, though.